kube/services/hatchery/hatchery-deploy.yaml

```diff
@@ -20,6 +20,9 @@ spec:
         public: "yes"
         netnolimit: "yes"
         userhelper: "yes"
+        tags.datadoghq.com/service: "hatchery"
+        tags.datadoghq.com/version: "master"
```

Comment:
We shouldn't hardcore version?

Reply:
I think this is just a temp default value and will get substituted when this guy kicks in? https://github.com/uc-cdis/cloud-automation/blob/master/gen3/lib/g3k_manifest.sh#L274

But I'm not 100% sure, my bash is rusty 😅

Reply:
Ah that makes sense!